README.md

```diff
@@ -25,113 +24,123 @@ Puppet Strings generates HTML documentation for Puppet extensions written in Pup
 
 ### Requirements
 
-To run Strings, you need the following software:
-
-  * Ruby 1.9.3 or newer
-  * Puppet 3.7 or newer
+  * Ruby 2.1.9 or newer
+  * Puppet 4.0 or newer
   * The `yard` Ruby gem
 
-Note that if you are running PE 3.8, you'll have a few extra steps to install puppet-strings.
+### Install Puppet Strings
 
-### 1. Install the YARD Gem
+1. Install the YARD gem by running `gem install yard`
+1. Install the `puppet-strings` gem by running `gem install puppet-strings`
+1. **Optional**: Set YARD options for Strings
+   
+   To use YARD options with Puppet Strings, specify a `yardopts` file in the same directory in which you run `puppet strings`. Puppet Strings supports the Markdown format and automatically sets the YARD `markup` option to `markdown`.
+   
+   To see a list of available YARD options, run `yard help doc`. For details about YARD options configuration, see the [YARD docs](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md#config).
 
-The easiest way to install the `yard` gem is with Puppet itself:
+## Generating documentation with Puppet Strings
 
-For Puppet >= 4 and Puppet Enterprise 2015.2 and later:
+By default, Puppet Strings outputs documentation as HTML, or you can specify JSON or Markdown output instead.
 
-```
-$ puppet resource package yard provider=puppet_gem
-```
+Strings generates reference documentation based on the code and Strings code comments in all Puppet and Ruby source files under the `./manifests/`, `./functions/`, and `./lib/` directories.
```

Comment:
./types/ , too. https://github.com/puppetlabs/puppet-strings/blob/master/lib/puppet-strings.rb#L5-L8

and ./tasks/ will be added before 2.0.0 as well.